src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1619,7 +1773,49 @@ There are two configuration options to control this mechanism (present in the `l
 ** The pa.ldap.tenant.attr configuration is not defined.
 ** The pa.ldap.tenant.attr configuration is defined, but the configured attribute is empty for the user.
 
-=== PAM
+[[_ldap_with_multiple_domains]]
+=== LDAP authentication with Multiple Domains
+
+The ProActive server can be configured to authenticate users according to *multiple LDAP domains*.
+
+Following is a step-by-step procedure to configure it.
+
+1. Copy the file `PROACTIVE_HOME\config\authentication\ldap.cfg` into multiple files, one for each domain.
++
+For example:
++
+----
+config\authentication\ldap-domain1.cfg
+config\authentication\ldap-domain2.cfg
+----
++
+2. Using the procedure described in chapter <<_ldap>>, configure each ldap file according to the domain used (each domain will have its own ldap server, bind login name, user subtree, etc.).
++
+3. Follow the steps described in chapter <<_select_authentication_method>> to configure `RMMultiLDAPLoginMethod` and `SchedulerMultiLDAPLoginMethod`.
++
+4. In `config\rm\settings.ini`, uncomment the property `pa.rm.multi.ldap.config` and define it according to the ldap configuration files you created and each domain name:
++
+----
+pa.rm.multi.ldap.config=domain1:config/authentication/ldap-domain1.cfg,domain2:config/authentication/ldap-domain2.cfg
+----
++
+5. In `config\scheduler\settings.ini`, uncomment the property `pa.scheduler.multi.ldap.config` and define it according to the ldap configuration files you created and each domain name:
++
+----
+pa.scheduler.multi.ldap.config=domain1:config/authentication/ldap-domain1.cfg,domain2:config/authentication/ldap-domain2.cfg
+----
+
+Once all the steps above are completed, the ProActive server can be started with a multi-domain ldap configuration.
```

Comment:
Once all the steps above are completed, the ProActive server can be (re)started  **in order to make the** multi-domain ldap configuration **effective**.